railties/test/application/loading_test.rb

```diff
@@ -116,7 +116,7 @@ class User < ActiveRecord::Base
     RUBY
 
     app_file "app/models/post.rb", <<-MODEL
-      class Post < ActiveRecord::Base
+      class Post < ApplicationRecord
```

Comment:
I changed this because technically in a new app this is what Post would inherit from and checking `defined?` loaded `ApplicationRecord` in this test. In master if I change this line the same thing happens so I think that this more accurately demonstrates what a real app would load/unload.